auth-api/src/auth_api/services/keycloak.py

```diff
@@ -277,3 +295,46 @@ def _get_group_id(admin_token: str, group_name: str):
     @staticmethod
     def _get_token_info():
         return g.jwt_oidc_token_info
+
+    @staticmethod
+    def _delete_user(user_id: str):
+        """Remove user from the keycloak."""
```

Comment:
may be write this method should only used for reset related functionalities..some other developer might misuse this for actually deleting the user

Reply:
I added a comment to notice other developers.